Chat_App/Chat_App/UI/MessageTableViewCell.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  MessageTableViewCell.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 01.05.23.
+//
+
+import UIKit
+
+final class MessageTableViewCell: UITableViewCell {
```

Comment:
Good point on using final before the class declaration to indicate that the class cannot be subclassed. It would be a good practice to use final for other classes as well